llvm/include/llvm/Passes/CodeGenPassBuilder.h

```diff
@@ -670,9 +670,6 @@ void CodeGenPassBuilder<Derived, TargetMachineT>::addIRPasses(
       !Opt.DisablePartialLibcallInlining)
     addPass(PartiallyInlineLibCallsPass());
 
-  // Instrument function entry and exit, e.g. with calls to mcount().
-  addPass(EntryExitInstrumenterPass(/*PostInlining=*/true));
```

Comment:
don't touch this one, this is the WIP port of the codegen pipeline to the new pass manager (and this is actually what we want, that the codegen pipeline does this)

Reply:
Reverted. Thanks for sharing the context about this file!